executor/common.h

```diff
@@ -470,6 +471,9 @@ static void loop(void)
 	}
 	for (i = 0; i < 100 && __atomic_load_n(&running, __ATOMIC_RELAXED); i++)
 		sleep_ms(1);
+#if SYZ_HAVE_CLOSE_FDS
+	close_fds();
```

Comment:
Does this close fd's in all the same cases we closed them before and in the same cases executor closes fd's?
This is only enabled if SYZ_THREADED, doesn't executor closes them even in non-threaded mode?

Reply:
Will be merged separately in #1112.